src/configurator/webpack-config/index.js

```diff
@@ -292,10 +309,16 @@ ${stylus}
     },
     'React hot loader': {
       group: 'React',
-      babel: babelConfig =>
-        Object.assign({}, babelConfig, {
+      babel: (babelConfig, configItems) => {
+        if (
+          _.includes(configItems, 'React hot loader') &&
+          !_.includes(configItems, 'Babel')
+        )
+          return {} // We don't need babelrc for typescript
```

Comment:
It seems to me like this change doesn't do anything. If babel is not selected, then `.babelrc` is not used anyway.

Reply:
It is still generating .babelrc if we select typescript and unselect babel, so this condition fixes that.